pkg/client/config/config.go

```diff
@@ -61,6 +61,9 @@ func RegisterFlags(fs *flag.FlagSet) {
 // If --kubeconfig is set, will use the kubeconfig file at that location.  Otherwise will assume running
 // in cluster and use the cluster provided kubeconfig.
 //
+// The returned `*rest.Config` has client-side ratelimting disabled as we can rely on API priority and
```

Comment:
Is it worth mentioning what the previous default configuration would have been for users who wish to restore/maintain the previous behaviour?

Reply:
IMHO no - We never documented the previous default either and I don't know why it existed

Reply:
We can mention it in release notes

Reply:
Ack, release notes is fine by me

Reply:
xref: https://github.com/kubernetes-sigs/controller-runtime/issues/3123